src/sentry/tasks/store.py

```diff
@@ -100,22 +100,25 @@ def should_demote_symbolication(project_id):
             "project_id": project_id,
         },
     )
-    return not never_lowpri and always_lowpri
 
+    if never_lowpri:
+        return False
+    elif always_lowpri:
+        return True
+    else:
+        return project_id in realtime_metrics.get_lpq_projects()
```

Comment:
We probably should add an O(1) method to `realtime_metrics` to check membership instead of this having to do the O(n) serialisation.  We will be running this for each event processed after all.